### PR TITLE
[WIP] Fix #134 - Add ltsc2019 and 1909 Windows images

### LINKED ISSUE #134: Provide Windows images for Windows 10, version 1909
There are Jenkins users who run their instances on GKE, and they may want to run Windows containers there as well. To do so, they need 1909 according to [this article](https://cloud.google.com/kubernetes-engine/docs/how-to/creating-a-cluster-windows?hl=fa):

_To run on GKE, Windows Server container node images need to be built on Windows Server version 2019 (LTSC) or Windows Server version 1909 (SAC). A single cluster can have multiple Windows Server node pools using different Windows Server versions, but each individual node pool can only use one Windows Server version._

CC @jtnord

### PR DESCRIPTION
**Pull request by slide**
This addresses #134 adding ltsc2019 (windowsservercore) and 1909 (windowsservercore and nanoserver) images. 

Also updated make.ps1 to search for dockerfiles within the directory structure so that it doesn't have to be updated when a new image is added.

### CODE REVIEW

**Comment by slide** [comment]
Bummer `docker : a Windows version 10.0.18363-based image is incompatible with a 10.0.17763 host`

**Comment by slide** [comment]
Moving this to WIP because we don't have the capability to build the images for ltsc2019 right now.

**Comment by slide** [comment]
Seems to only be the nanoserver-1909 image that fails to build now.

File: 11/windows/windowsservercore-1809/Dockerfile [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
+    Remove-Item -Force (Join-Path C:/ProgramData/Jenkins $env:AGENT_HASH_FILENAME)
+
+USER $user
+
+RUN New-Item -Type Directory $('{0}/.jenkins' -f $env:AGENT_ROOT) | Out-Null ; `
+    New-Item -Type Directory $env:AGENT_WORKDIR | Out-Null
+
+VOLUME ${AGENT_ROOT}/.jenkins
+VOLUME ${AGENT_WORKDIR}
+WORKDIR ${AGENT_ROOT}
```
Comment (olblak) [inline 11/windows/windowsservercore-1809/Dockerfile:82]: Add newline at the end of the file
```suggestion
WORKDIR ${AGENT_ROOT}

```

Comment (olblak) [inline 11/windows/windowsservercore-1809/Dockerfile:82]: Missing newline

**Pull request closed without merging**